src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write code here
+const head = document.querySelector('thead');
+const body = document.querySelector('tbody');
+const bodychildren = [...body.children];
+
+function transformationNumber(element) {
```

Comment:
```suggestion
function transformToNumber(element) {
```
it's better to name functions in this way :arrow_up: (when it contains **verb** that explains what the function is doing)